opmon-protobuf-dbwriter/dbwriter.py

```diff
@@ -0,0 +1,168 @@
+# @file dbwriter.py Writing Opmon entries into InfluxDB
+#  This is part of the DUNE DAQ software, copyright 2020.
+#  Licensing/copyright details are in the COPYING file that you should have
+#  received with this code.
+#
+
+import kafkaopmon.OpMonSubscriber as opmon_sub
+import google.protobuf.json_format as pb_json
+from google.protobuf.timestamp_pb2 import Timestamp
+import opmonlib.opmon_entry_pb2 as opmon_schema
+
+from influxdb import InfluxDBClient
+from functools import partial
+import json
+import click
+import logging
+import queue
+import threading
+
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+# subscriber options
+@click.option('--subscriber-bootstrap', type=click.STRING,                default="monkafka.cern.ch:30092", help="boostrap server and port of the OpMonSubscriber")
+@click.option('--subscriber-group',     type=click.STRING,                default=None, help='group ID of the OpMonSubscriber')
+@click.option('--subscriber-timeout',   type=click.INT,                   default=500, help='timeout in ms used in the OpMonSubscriber')
+@click.option('--subscriber-topic',     type=click.STRING, multiple=True, default=['opmon_stream'], help='The system will add the "monitoring." prefix' )
+
+#influx options
+@click.option('--influxdb-address', type=click.STRING, default='np04-srv-017', help='address of the influx db')
```

Comment:
Should this be `np04-srv-017.cern.ch`?

Reply:
In fact I came put monkafka.cern.ch so that we make things more uniform to the other microservices.